blackbox/docs/sql/general/lexical-structure.rst

```diff
@@ -157,6 +157,8 @@ the SQL standards from SQL-92 to SQL-2011.
     DEFERRED, LOWER, SAVEPOINT,
     DELETE, MAP, SCHEMA,
     DENSE_RANK, MATCH, SCOPE,
+    WORK, SERIALIZABLE, REPEATABLE, COMMITTED
```

Comment:
isn't this a breaking change as we have new keywords that need quoting?

Reply:
you're right. however, these new keywords are non-reserved keywords, they don't require quoting. 
from looking at this list, I think hardly any of them really need to be quoted.

e.g.
```
cr> create table t1 (lower string, decimal string, work string, savepoint string);
CREATE OK, 1 row affected  (1.952 sec)
cr> select lower, decimal, work from t1;
+-------+---------+------+
| lower | decimal | work |
+-------+---------+------+
+-------+---------+------+
SELECT 0 rows in set (0.005 sec)
```

Guess we need to go through this list and check every single keyword in this list again ....

Reply:
removed these keywords, as they are non-reserved